rules/windows/lateral_movement_cmd_service.toml

```
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/09/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of sc.exe to create, modify, or start services on remote hosts. This could be indicative of adversary
+lateral movement but will be noisy if commonly done by admins.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Service Command Lateral Movement"
+risk_score = 21
+rule_id = "d61cbcf8-1bc1-4cff-85ba-e7b21c5beedc"
+severity = "low"
+tags = ["Elastic", "Windows"]
+type = "eql"
+
+query = '''
+// dependent on a wildcard for remote path
+
+sequence by process.entity_id, process.pid, process.name with maxspan=1m
+  [process where event.type in ("start", "process_started") and
+     (process.name == "sc.exe" or process.pe.original_file_name == "sc.exe") and
+     process.args == "\\\\*" and process.args == "binPath*" and
```

until we have `:`, we can do this for case-sensitive
```suggestion
     wildcard(process.args, "\\\\*") and wildcard(process.args, "binPath*", binpath*") and
```